app/controllers/invites_controller.rb

```diff
@@ -361,6 +361,12 @@ def resend_invite
   def resend_all_invites
     guardian.ensure_can_resend_all_invites!(current_user)
 
+    begin
```

Comment:
Bulk invite sending/resending is only available to staff, so the regular user values are not used here. Maybe we can simplify this a bit, and pass a simple `apply_to_staff` flag to the rate limiter and if present, the rate limiter applies using the existing `max` and `secs` values?

Reply:
Also, it would be nice to have a spec for rate limiting in the `invites_controller_spec`.

Reply:
> Bulk invite sending/resending is only available to staff, so the regular user values are not used here.

From my understanding, the ability to send invites can be set by trust level, so in theory this could affect non staff users.

Reply:
I think individual invites can be sent by TLs. But bulk sending goes through this: 

https://github.com/discourse/discourse/blob/b324113f635ccf62172a9d55860756288c1e790f/lib/guardian.rb#L405-L407

Reply:
Doh, thank you. I will update this RL with the simpler flag, and clean this up

Reply:
Renamed